### PR TITLE
fix: replace inplayer integration key with jwp

### PR DESCRIPTION
**Pull request by budzworthy**
## Description

Remove the inplayer key from integrations and just use jwp.

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: src/hooks/useClientIntegration.ts [hunk 1]
```diff
@@ -10,12 +10,11 @@ const useClientIntegration = () => {
...
-  const isJWP = !!integrations.jwp?.clientId || !!integrations.inplayer?.clientId;
-  const jwpIntegration = integrations.jwp?.clientId ? integrations.jwp : integrations.inplayer;
-  const sandbox = isJWP ? jwpIntegration?.useSandbox : integrations.cleeng?.useSandbox;
+  const isJWP = !!integrations.jwp?.clientId;
+  const sandbox = isJWP ? integrations.jwp?.useSandbox : integrations.cleeng?.useSandbox;
   const client = isJWP ? ClientIntegrations.JWP : ClientIntegrations.CLEENG;
-  const clientId = isJWP ? jwpIntegration?.clientId : integrations.cleeng?.id;
-  const clientOffers = isJWP ? [`${jwpIntegration?.assetId}`] : [`${integrations.cleeng?.monthlyOffer}`, `${integrations.cleeng?.yearlyOffer}`];
+  const clientId = isJWP ? integrations.jwp?.clientId : integrations.cleeng?.id;
+  const clientOffers = isJWP ? [`${integrations.jwp?.assetId}`] : [`${integrations.cleeng?.monthlyOffer}`, `${integrations.cleeng?.yearlyOffer}`];
```
Comment (AntonLantukh) [inline src/hooks/useClientIntegration.ts:17]: Do we need to use template strings here?

File: src/utils/configLoad.ts [hunk 2]
```diff
@@ -37,23 +37,20 @@ const maybeInjectAnalyticsLibrary = (config: Config) => {
 };
 
 const calculateAccessModel = (config: Config): AccessModel => {
-  if (config?.integrations?.cleeng) {
-    const { id, monthlyOffer, yearlyOffer } = config?.integrations?.cleeng || {};
+  if (config?.integrations?.cleeng?.id) {
+    if (config?.integrations?.cleeng?.monthlyOffer || config?.integrations?.cleeng?.yearlyOffer) {
```
Comment (AntonLantukh) [inline src/utils/configLoad.ts]: Probably ternary operator could simplify it for cleeng and inplayer.

```
const {cleeng, jwp} = config?.integrations || {};

if (cleeng?.id) {
    return cleeng?.monthlyOffer || cleeng?.yearlyOffer ? 'SVOD' : 'AUTHVOD';
}

if (jwp?.clientId)  {
    return jwp?.assetId ? 'SVOD' : 'AUTHVOD';
}

return 'AVOD';

```

File: src/utils/configLoad.ts [hunk 3]
```diff
@@ -112,11 +109,11 @@ export async function loadAndValidateConfig(configSource: string | undefined) {
 
   maybeInjectAnalyticsLibrary(config);
```
Comment (AntonLantukh) [inline src/utils/configLoad.ts:103]: const {cleeng, jwp} = config?.integrations || {};

**Review by AntonLantukh — APPROVED** [review]

File: src/utils/configLoad.ts (hunk 3 above)
Comment (budzworthy) [reply src/utils/configLoad.ts:103]: What does this comment go to?

File: src/hooks/useClientIntegration.ts (hunk 1 above)
Comment (budzworthy) [reply src/hooks/useClientIntegration.ts:17]: No idea, I just left it as is....

File: src/utils/configLoad.ts [hunk 4]
```diff
@@ -112,11 +109,11 @@ export async function loadAndValidateConfig(configSource: string | undefined) {
...
 
-  // TODO: refactor this once we have more input how integrations will be handled in dashboard
-  if (config?.integrations?.cleeng?.id && (config?.integrations?.jwp?.clientId || config?.integrations?.inplayer?.clientId)) {
+  if (config?.integrations?.cleeng?.id && config?.integrations?.jwp?.clientId) {
```
Comment (AntonLantukh) [inline src/utils/configLoad.ts:104]: @dbudzins we could use it here

```
const {cleeng, jwp} = config?.integrations || {};

  if (cleeng?.id && jwp?.clientId) {
  ...
```

File: src/utils/configLoad.ts (hunk 3 above)
Comment (AntonLantukh) [reply src/utils/configLoad.ts:103]: Left one more comment with a better location

File: src/utils/configLoad.ts (hunk 2 above)
Comment (budzworthy) [reply src/utils/configLoad.ts]: Nice, updated

File: src/utils/configLoad.ts (hunk 4 above)
Comment (budzworthy) [reply src/utils/configLoad.ts:104]: I tend to err on the side of fewer temp variables where possible. The chaining doesn't bother me in this case.

**Review by AntonLantukh — APPROVED** [review]

**Pull request merged**